test/test_automl/test_estimators.py

```diff
@@ -317,6 +320,125 @@ def test_cv_results(tmp_dir):
     assert hasattr(cls, 'classes_')
 
 
+@pytest.mark.parametrize('estimator_type,dataset_name', [
+    (AutoSklearnClassifier, 'iris'),
+    (AutoSklearnRegressor, 'boston')
+])
+def test_leaderboard(
+    tmp_dir: str,
+    estimator_type: Type[AutoSklearnEstimator],
+    dataset_name: str
+):
+    # Comprehensive test tasks a substantial amount of time, manually set if
+    # required.
+    MAX_COMBO_SIZE_FOR_INCLUDE_PARAM = 2  # [0, len(valid_columns) + 1]
+
+    X_train, Y_train, _, _ = putil.get_dataset(dataset_name)
+    model = estimator_type(
+        time_left_for_this_task=30,
+        per_run_time_limit=5,
+        tmp_folder=tmp_dir,
+        seed=1
+    )
+    model.fit(X_train, Y_train)
+
+    valid_columns = [
```

Comment:
Could these be put as constants somewhere so that you don't have to redefine them here?